app/controllers/search_results_controller.rb

```diff
@@ -0,0 +1,45 @@
+class SearchResultsController < ApplicationController
+  before_action :set_search_result, only: %i[show edit update destroy]
+
+  # GET /search_results
+  def index
+    @search_results ||= []
+
+    10.times do |number|
+      @search_results.append(SearchResult.new(
+                               id: number,
+                               title: format("Search Result %d", number),
+                               link: search_results_path
+                             ))
+    end
+    @search_results = @search_results.uniq(&:id)
+  end
+
+  def create
+    @search_result = SearchResult.new(search_result_params)
+  end
+
+  # There is no need to display one singular SearchResult.
+  def show; end
+
+  # SearchResults are currently not stored a database so no need to edit one.
+  def edit; end
+
+  # SearchResults are currently not stored a database so no need to update one.
+  def update; end
+
+  # All SearchResult objects will be destroyed after they are no longer used.
+  def destroy; end
+
+  private
+
+  # Use callbacks to share common setup or constraints between actions.
+  def set_search_result
+    @search_result = SearchResult.find(params[:id])
+  end
+
+  # Only allow a list of trusted parameters through.
+  def search_result_params
+    params.fetch(:search_result, {})
```

Comment:
Use `require` and `permit` to define which parameters should be accepted. More Information can be found [here](https://guides.rubyonrails.org/v6.0/getting_started.html#saving-data-in-the-controller).